demo/sdk/CMakeLists.txt

```diff
@@ -10,19 +10,30 @@ file(GLOB_RECURSE EXTRA_CPP_BINDINGS include/extra-cpp-bindings/src/*.cc)
 file(GLOB PLAY_CPP_SDK_SROUCES include/*.cc)
 
 # Find the rust types binding library
-find_library(RUST_LIB cxxbridge1 REQUIRED PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
 
 # Find the prebuilt static and dynamic libraries
-if (WIN32)
+if (WIN32 AND MSYS)
+  find_library(RUST_LIB cxxbridge1 REQUIRED PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
+  find_library(PLAY_CPP_SDK_LIB libplay_cpp_sdk.a PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
+  find_library(PLAY_CPP_SDK_DYLIB libplay_cpp_sdk.dll.a PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
+  find_file(PLAY_CPP_SDK_DLL NAME play_cpp_sdk.dll PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
+  # Add static library play_cpp_sdk built from bindings source files
+  add_library(play_cpp_sdk STATIC ${DEFI_WALLET_CORE_CPP_BINDINGS} ${EXTRA_CPP_BINDINGS} ${PLAY_CPP_SDK_SROUCES})
+  target_link_libraries(play_cpp_sdk ${PLAY_CPP_SDK_LIB} Ncrypt userenv ntdll Secur32 crypt32 ws2_32 ntdll)
+endif()
+
+if (WIN32 AND NOT MSYS)
+  find_library(RUST_LIB cxxbridge1 REQUIRED PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
   find_library(PLAY_CPP_SDK_LIB play_cpp_sdk.lib PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
   find_library(PLAY_CPP_SDK_DYLIB play_cpp_sdk.dll.lib PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
   find_file(PLAY_CPP_SDK_DLL NAME play_cpp_sdk.dll PATHS ${CMAKE_CURRENT_SOURCE_DIR}/lib)
   # Add static library play_cpp_sdk built from bindings source files
   add_library(play_cpp_sdk STATIC ${DEFI_WALLET_CORE_CPP_BINDINGS} ${EXTRA_CPP_BINDINGS} ${PLAY_CPP_SDK_SROUCES})
-  target_link_libraries(play_cpp_sdk ${RUST_LIB} ${PLAY_CPP_SDK_DYLIB} Ncrypt userenv ntdll Secur32 crypt32)
+  target_link_libraries(play_cpp_sdk ${PLAY_CPP_SDK_LIB} Ncrypt userenv ntdll Secur32 crypt32 ws2_32 ntdll)
```

Comment:
sdk is a individual folder that could be move to other projects. We have to link RUST_LIB in sdk folder.

Reply:
yes, i'll check